File: samples/demo-projects-python/project-a/calculator.py
class Calculator:
    def __init__(self, value):
        self.value = value

    def add(self, number):
        result = self.value + number
        return result

    def subtract(self, number):
        result = self.value - number
        return result

    def multiply(self, factor):
        total = 0
        for i in range(factor):
            total = total + self.value
        return total

    def is_positive(self):
        if self.value > 0:
            return True
        else:
            return False

File: samples/demo-projects-js/project-a/calculator.js
class Calculator {
  constructor(value) {
    this.value = value;
  }

  add(number) {
    const result = this.value + number;
    return result;
  }

  subtract(number) {
    const result = this.value - number;
    return result;
  }

  multiply(factor) {
    let total = 0;
    for (let i = 0; i < factor; i = i + 1) {
      total = total + this.value;
    }
    return total;
  }

  isPositive() {
    if (this.value > 0) {
      return true;
    } else {
      return false;
    }
  }
}
